app/page.tsx

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) ZeroC, Inc.
+
+import React from 'react';
+import { Hero } from './hero';
+import { HomeTitle } from 'components/Tags/HomeTitle';
+import { AppLink, Divider, Table, TD, TH, TR } from 'components';
+import { Explore } from 'components/Tags/Explore';
+import { Examples } from 'components/Tags/Examples';
+import { License } from 'components/Tags/License';
+
+export default async function Home() {
+  return (
+    <div className="mt-[3.75rem] flex grow flex-row justify-center lg:mt-[3.75rem]">
+      <div className="relative flex max-w-[120rem] grow flex-col items-center justify-center">
+        <div id="skip-nav" />
+        <Hero />
+        <div className="flex shrink flex-row justify-center overflow-y-clip lg:justify-start">
+          <article className="mx-6 mb-20 mt-10 h-full w-full max-w-[52rem] md:mx-10 lg:mx-16">
+            <HomeTitle title="Welcome" description="" />
+            <p>
+              Welcome to the IceRPC documentation! You will find on this site a
+              wide range of materials, including installation instructions,
+              tutorials, programming guides, and more.
+            </p>
+            <p>
+              Want to make RPCs without reinventing the wheel? Want to take
+              advantage of QUIC through a convenient, modern API? You&apos;ve
+              come to the right place.
+            </p>
+
+            <p>
+              IceRPC is currently available for C# / .NET. We plan on adding
+              support for more languages soon, starting with Rust.
+            </p>
+            <Divider />
+            <HomeTitle title="Key Features" description="Discover IceRPC" />
```

Comment:
I actually prefer the site without this, but we used to have this line above the key features:
"Jump directly to the documentation of these distinctive features:"
Like I said, I'm fine dropping it, but just want to make sure it wasn't dropped unintentionally.
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/faee6b81-fd6e-452e-b796-6998b77a582e)

Reply:
Was unintentional. Trying to keep this PR as close to what the current site looks like as possible. In the future we could open separate PRS for stuff like this.